cypress/support/components/entitypage.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+
+/** Navigate to the Entities page */
+function NavigateTo() {
+    cy.get('a[href$="/entities"]').click().wait(1000)
```

Comment:
Find an alternative rather than `wait(1000)` or explain why you can't.  This is not very robust.